S3 London. All good with updated tests.

Failure in `ITestAssumeRole` which seems unrelated (will need to test on base branch-3.3)
{code}

[ERROR]   ITestAssumeRole.testAssumeRoleBadInnerAuth:263->expectFileSystemCreateFailure:137  Expected to find 'not a valid key=value pair (missing equal-sign) in Authorization header' but got unexpected exception: org.apache.hadoop.fs.s3a.AWSBadRequestException: Instantiate org.apache.hadoop.fs.s3a.auth.AssumedRoleCredentialProvider: com.amazonaws.services.securitytoken.model.AWSSecurityTokenServiceException: Invalid key=value pair (missing equal-sign) in Authorization header (hashed with SHA-256 and encoded with Base64): 'hvHJSzfhtkBgtrEOkT9w/UT3So5LWUnDLLsTKYln5vE='. (Service: AWSSecurityTokenService; Status Code: 400; Error Code: IncompleteSignature; Request ID: 4f76f95a-3933-4312-858b-3f1d123bf14f; Proxy: null):IncompleteSignature: Invalid key=value pair (missing equal-sign) in Authorization header (hashed with SHA-256 and encoded with Base64): 'hvHJSzfhtkBgtrEOkT9w/UT3So5LWUnDLLsTKYln5vE='. (Service: AWSSecurityTokenService; Status Code: 400; Error Code: IncompleteSignature; Request ID: 4f76f95a-3933-4312-858b-3f1d123bf14f; Proxy: null)                                                                                                                                                     
{code}

--- a/hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/impl/InternalConstants.java
+++ b/hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/impl/InternalConstants.java
@@ -127,14 +127,12 @@ public final class InternalConstants {
   public static final int SC_403_FORBIDDEN = 403;
 
   /** 403 error code. */
-  @Deprecated
   public static final int SC_403 = SC_403_FORBIDDEN;
 
   /** 404 status code: Not Found. */
   public static final int SC_404_NOT_FOUND = 404;
 
   /** 404 error code. */
-  @Deprecated
   public static final int SC_404 = SC_404_NOT_FOUND;
 
   /** 405 status code: Method Not Allowed. */

